katello/hooks/pre/11-clear_pulpcore_data_feature.rb

```diff
@@ -0,0 +1,12 @@
+# See bottom of the script for the command that kicks off the script
+
+def clear_pulpcore_content
+  if File.directory?('/var/lib/pulp/docroot')
+    `rm -rf /var/lib/pulp/docroot/*`
+    logger.info 'Pulpcore content successfully removed.'
+  else
+    logger.warn 'Pulpcore content directory not present at \'/var/lib/pulp/docroot\''
+  end
+end
+
+clear_pulpcore_content if app_value(:clear_pulpcore_content) && !app_value(:noop)
```

Comment:
Could you add a `module_enabled?('foreman_proxy_content')` condition to the if check

Reply:
do you think clear_pulp_data_feature should have this too?